dockers/base-xla/Dockerfile

```diff
@@ -92,6 +92,10 @@ RUN \
     python --version && \
     cd pytorch-lightning && \
     pip install -q fire && \
+    # Pin mkl version to avoid OSError on torch import
+    # OSError: libmkl_intel_lp64.so.1: cannot open shared object file: No such file or directory
+    # https://github.com/pytorch/xla/issues/1666
+    pip install mkl==2021.4.0 && \
```

Comment:
@akihironitta Why did you pin to this specific release?

I am still seeing the `OSError` in https://github.com/Lightning-AI/lightning/pull/14926. I'll try unpinning

Reply:
If I remember correctly, that version was the latest release that worked in our CI at the time.